test/plugin.test.js

```diff
@@ -337,3 +337,21 @@ QUnit.test('custom error details should override defaults', function(assert) {
   assert.strictEqual(document.querySelector('.vjs-errors-message').textContent,
     customError.message, 'message should match custom override value');
 });
+
+QUnit.test('Append Flash error details when flash is not supported', function(assert) {
+  let isFlashSupported = videojs.getComponent('Flash').isSupported();
+
+  if (!isFlashSupported) {
```

Comment:
@BrandonOCasey Do we want to append the flash message if flash is supported ?
Don't we want the test to only run when flash is not supported ? 
Correct me if I'm missing something, don't we want to only append the flash message when flash is either not installed or not supported.